### ISSUE TITLE
Setting weekly workout.

### ISSUE THREAD

**Issue by Zaki5581** [Zaki5581/capstone-project#3]
## Value proposition

**As a** user,
**I want to be able to save my workouts.
**so that  i can edit or delete them in the future .

## Description

<img width="398" alt="Bildschirmfoto 2022-06-24 um 09 31 49" src="https://user-images.githubusercontent.com/92207171/175486191-9f05cb75-4e94-43fe-9654-7d30490f180d.png">

## Acceptance criteria

-[ ]after clicking a body part, just exercises for this body part should be displayed.
-[ ]if there are many workouts ,they should fit properly in the workouts container. 

## Complexity

 Large

## Tasks

-[ ] Create branch.
-[ ]Add base styles.
-[ ]Add new inputs for body part and day.
-[ ]set state for exercise and plan.
-[ ]render the workouts.

**Comment by AnnaGrasm** [comment]
Hey Zakaria, langsam kommst du mit deiner App. Das freut mich. Ich finde, dass die Überschrift "Setting weekly workout" irgendwie nicht mit der UserStory passt. Bei den Acceptance criteria klingt es eher so, als ob du eher filtern möchtest. Oder willst du local storage einbauen wegen "As a user, I want to be able to save my workouts? ":)

**Comment by Zaki5581** [comment]
Ja, was auf dem button steht ist nicht unbedingt gut, ja mit localstorage werde ich versuchen in dieser story erst nur vorzubereiten, damit es in der nächsten story mit localstorage losgeht.
